cabal.project

```diff
@@ -14,38 +14,39 @@ repository cardano-haskell-packages
 -- We duplicate the hackage index-state first for haskell.nix, even though cabal ignores it.
 -- This must always match the hackage index-state on the next line.
 -- See https://github.com/input-output-hk/haskell.nix/issues/1869#issuecomment-1449272480
-index-state: 2023-05-26T00:00:00Z
+index-state: 2023-11-14T00:00:00Z
 index-state:
-  , hackage.haskell.org      2023-05-26T00:00:00Z
-  , cardano-haskell-packages 2023-05-26T00:00:00Z
+  , hackage.haskell.org      2023-11-14T00:00:00Z
+  , cardano-haskell-packages 2023-11-14T00:00:00Z
 
+-- FIXME: This was commented out for PLT-8148, but needs to be restore before that branch is merged to main.
```

Comment:
These will be uncommented when the relevant packages are upgraded, but it doesn't make sense to do that until the production version of `cardano-node` uses Plutus v1.15.0.0.